crates/goose/src/session/storage.rs

```diff
@@ -158,6 +158,14 @@ pub fn get_path(id: Identifier) -> Result<PathBuf> {
             session_dir.join(format!("{}.jsonl", name))
         }
         Identifier::Path(path) => {
+            // Allow special paths for no-session mode
+            if let Some(path_str) = path.to_str() {
+                if path_str == "/dev/null" || path_str == "NUL" {
```

Comment:
why would it be NUL? that doesn't sound right

Reply:
windows
![image](https://github.com/user-attachments/assets/bbfd751b-e703-47e3-8cb6-d4143d5cd6f3)